hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/IncrementalTableBackupClient.java

```diff
@@ -489,7 +495,7 @@ private void verifyCfCompatibility(Set<TableName> tables,
         try {
           fs = FileSystem.get(new URI(fullBackupInfo.getBackupRootDir()), conf);
         } catch (URISyntaxException e) {
-          throw new IOException("Unable to get fs", e);
+          throw new IOException("Unable to get fs for backup " + fullBackupInfo.getBackupId(), e);
```

Comment:
Just addressing [this](https://github.com/apache/hbase/pull/6340#discussion_r1802617631) comment, while I'm here.